app/models/practice_resource.rb

```diff
@@ -0,0 +1,30 @@
+class PracticeResource < ApplicationRecord
+  acts_as_list scope: :practice
+  after_create :attachment_crop
```

Comment:
I think in a previous convo you said you removed the crop related code but it looks like it is still here 😅

Reply:
Yeah - sorry I meant to tell you...when I removed that code... I ran in to issues saving the attachments.  I'll try it again but as soon as I put the code back in.. it was working.

Reply:
Good catch on the line seperator... i see it's only "Within' each section not between.  :)  added the line separator.